src/datepicker/datepicker-input.ts

```diff
@@ -185,6 +185,11 @@ export class NgbInputDatepicker implements OnChanges,
    */
   @Output() navigate = new EventEmitter<NgbDatepickerNavigateEvent>();
 
+  /**
+   * An event fired after closing datepicker window.
+   */
+  @Output() closed = new EventEmitter();
```

Comment:
Please add type `new EventEmitter<void>()`

Reply:
Thanks for advice, changes has been done.
Actually I've got a question, is
`closed: EventEmitter<void> = new EventEmitter()`
equal to
`closed = new EventEmitter<void>()`
will Generic Type be forced to void in second case?

Reply:
Basic type inference is described in → https://www.typescriptlang.org/docs/handbook/type-inference.html. No generics there though...

Anyway looks like it works, at least autocompletion in IDE understands that it is specifically `EventEmitter<void>` and if you try `this.close.emit(true);`, it will complain.

There is also a tslint rule called "no-inferrable-types" → https://palantir.github.io/tslint/rules/no-inferrable-types/. And we have it active, but it works only for simple types like `const b: boolean = false`